UEFITool/uefitool.cpp

```diff
@@ -858,6 +934,43 @@ void UEFITool::openImageFile(QString path)
     currentPath = path;
 }
 
+void UEFITool::specifyPathIDA()
+{
+    specifyPathIDA32();
+    specifyPathIDA64();
+}
+
+void UEFITool::specifyPathIDA32()
+{
+    UString path;
+
+#if defined Q_OS_WIN
+    path = QFileDialog::getOpenFileName(this, tr("Specify path to ida.exe"),  "ida.exe", tr("IDA Pro executable (ida.exe);;All files (*)"));
+#else
+    path = QFileDialog::getOpenFileName(this, tr("Specify path to ida.exe"),  "ida.exe", tr("IDA Pro executable (ida);;All files (*)"));
```

Comment:
I think it is still a little messed up in the 2nd and 3rd arguments. Same below.